controllers/argocd/notifications.go

```diff
@@ -432,6 +454,62 @@ func (r *ReconcileArgoCD) reconcileNotificationsDeployment(cr *argoproj.ArgoCD,
 
 }
 
+// reconcileNotificationsService will ensure that the Service for the Notifications controller metrics is present.
+func (r *ReconcileArgoCD) reconcileNotificationsService(cr *argoproj.ArgoCD) error {
+
+	var component = "notifications-controller"
+	svc := newServiceWithSuffix(component, component, cr)
```

Comment:
I think metrics related resources should have the `-metrics` suffix in the name so its easy to distinguish them WDYT?

Reply:
But in that case, all the service monitors will be created with the same name.

Reply:
not sure  I follow
each svc monitor should be qualified by instance name, component and suffix so no 2 svc monitors should collide on the name

Reply:
```suggestion
        var component = "notifications-controller"
        var suffix = "notifications-controller-metrics"
	svc := newServiceWithSuffix(suffix, component, cr)
```